src/app/fsat/operations/operations.component.ts

```diff
@@ -0,0 +1,176 @@
+import { Component, ElementRef, EventEmitter, HostListener, Input, OnInit, Output, SimpleChanges, ViewChild } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FieldData, FSAT, FsatOperations } from '../../shared/models/fans';
+import { OperatingHours } from '../../shared/models/operations';
+import { Settings } from '../../shared/models/settings';
+import { CompareService } from '../compare.service';
+import { FanOperationsWarnings, FsatWarningService } from '../fsat-warning.service';
+import { FsatService } from '../fsat.service';
+import { HelpPanelService } from '../help-panel/help-panel.service';
+import { OperationsService } from './operations.service';
+
+@Component({
+  selector: 'app-operations',
+  templateUrl: './operations.component.html',
+  styleUrls: ['./operations.component.css']
+})
+export class OperationsComponent implements OnInit {
+  @Input()
+  fsat: FSAT;  
+  @Input()
+  fsatOperations: FsatOperations;
+  @Input()
+  settings: Settings;
+  @Output('emitSave')
+  emitSave = new EventEmitter<FsatOperations>();
+  @Input()
+  selected: boolean;
+  @Input()
+  inSetup: boolean;
+  @Input()
+  baseline: boolean;
+  @Input()
+  modificationIndex: number;
+
+  @ViewChild('modalBody', { static: false }) public modalBody: ElementRef;
+
+  @ViewChild('formElement', { static: false }) formElement: ElementRef;
+  @HostListener('window:resize', ['$event'])
+  onResize(event) {
+    this.setOpHoursModalWidth();
+    this.getBodyHeight();
+  }
+
+  warnings: FanOperationsWarnings;
+  
+  formWidth: number;
+  
+  bodyHeight: number;
```

Comment:
bodyHeight unused, can remove variable and getBodyHeight function

theres some other stuff in this component that are unused but left over from copy/pasting